software-mansion/react-native-screens
# feat: add preload support

## Description

Adds support for using `CommonActions.preload`.
Also can be used to for iOS PiP https://github.com/software-mansion/react-native-screens/pull/1665.

Corresponding PR in `react-navigation`:

* https://github.com/react-navigation/react-navigation/pull/12083

## Changes

- Added prop `hiddenFromStack`

## Test code and steps to reproduce

1. Run new test `TestPreload`.
2. Tap on preload buttons and check if the preload time increases.

## Checklist
